lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -28,6 +28,7 @@ const (
 	Imagec           = "imagec"
 	Retries          = 5
 	RetryTimeSeconds = 2
+	PortlayerName    = "Backend Engine"
```

Comment:
Given this is used to identify the graph driver that's associated with an image, "vSphere" is a better user facing term.